HARK/ConsumptionSaving/ConsAggShockModel.py

```diff
@@ -1725,9 +1728,8 @@ def calcAFunc(self, MaggNow, AaggNow):
         for i in range(self.MrkvArray.shape[0]):
             these = i == MrkvHist
             slope, intercept, r_value, p_value, std_err = stats.linregress(logMagg[these], logAagg[these])
-            if verbose:
-                #plt.plot(logMagg[these],logAagg[these],'.')
-                pass
+            if hasattr(self, 'plot_verbose'):
```

Comment:
now the PR inconsistently uses 'plot_verbose' or 'verbose_plot' for this feature.

I'm thumbs down on:
 - adding undocumented model parameters
 - the model specific controls of verbosity, discussed in #701

But if this is necessary functionality, then maybe we can wire it through the "view" code in a way that's clean.